zulipterminal/ui_tools/views.py

```diff
@@ -1126,19 +1133,31 @@ def __init__(
         no._w = urwid.AttrMap(urwid.SelectableIcon("No", 4), None, "selected")
         display_widget = urwid.GridFlow([yes, no], 3, 5, 1, "center")
         wrapped_widget = urwid.WidgetWrap(display_widget)
-        prompt = urwid.LineBox(
-            urwid.ListBox(
-                urwid.SimpleFocusListWalker([question, urwid.Divider(), wrapped_widget])
-            )
-        )
+        widgets = [question, urwid.Divider(), wrapped_widget]
+        prompt = urwid.LineBox(urwid.ListBox(urwid.SimpleFocusListWalker(widgets)))
```

Comment:
A good follow-up to this would be to improve the styling and unify it with the general popup - the border is not that clear compared to those, in retrospect.